### ISSUE TITLE
checking if pages have logical tab order

### ISSUE THREAD

**Issue by a20behta** [HGustavs/LenaSYS#11889]
related to #11794 

check each tab if they follow logica order.
first point in manual check accessability

**Comment by b20hanwa** [comment]
**This comment applies to the first manual check on the list:**
<img width="326" alt="Accessability8" src="https://user-images.githubusercontent.com/81630271/165745010-1b2302df-2b7d-42ca-8919-32852ca733c9.png">

When tabbing through LenaSYS I came to understand that a lot of the interactive buttons on the page are not accessible via keyboard navigation. The buttons marked with red in the images below have this problem:


<img width="960" alt="Accessability1" src="https://user-images.githubusercontent.com/81630271/165745773-372fd774-36b6-4135-87a7-86cff5d09087.png">


<img width="960" alt="Accessability2" src="https://user-images.githubusercontent.com/81630271/165745796-b627b262-ddd5-468e-aa33-006dc52c45d3.png">


<img width="949" alt="Accessability3" src="https://user-images.githubusercontent.com/81630271/165745819-10db942b-c4ff-4791-aa9a-a15ddfb0510c.png">

When having a pop-up on the page, you first have to tab through the page in the background to reach the pop-up. The page in focus (in this case; the pop-up) should be the one that you can tab through. 


<img width="948" alt="Accessability4" src="https://user-images.githubusercontent.com/81630271/165745909-d484008f-84bb-4313-86c4-518a2ff344a3.png">


<img width="949" alt="Accessability5" src="https://user-images.githubusercontent.com/81630271/165746000-43cbd812-f21d-48a8-af01-1c3c1cc8c384.png">


<img width="963" alt="Accessability6" src="https://user-images.githubusercontent.com/81630271/165746058-b691e596-5fdf-4d59-be2a-ea900d5b3d06.png">


<img width="950" alt="Accessability7" src="https://user-images.githubusercontent.com/81630271/165746118-3f6d13e6-74e2-4214-9969-180f348e947a.png">

**Comment by b20hanwa** [comment]
**This comment also applies to the first manual check on the list:** 
<img width="326" alt="Accessability8" src="https://user-images.githubusercontent.com/81630271/165748790-c7535808-b5cf-4691-9f6d-745381c7d721.png">


The keyboard navigation seems to work quite well on the Contribution page. Only one interactive alement is not possible to navigate to:

<img width="959" alt="Accessability1 1" src="https://user-images.githubusercontent.com/81630271/165749075-1495aac8-595b-427c-9aee-f67511098b22.png">

**Comment by b20hanwa** [comment]
This is what Lighthouse suggest as solution for the tab order on a page:

<img width="697" alt="Accessability9" src="https://user-images.githubusercontent.com/81630271/165749597-2458ec3c-4852-4238-a68a-da8796043f73.png">

**Comment by a20vicad** [comment]
'Account requests' is now able to be opened and closed with keybord. Issue: #12067
